# Project 1 -- Arnav Surve

**TA Help:** N/A

**Collaboration:** N/A

## Question 1

In [1]:
# Print the sum of 7 and 10
print(7+10)

17


## Question 2

In [2]:
x = 6
y = 8
z = x * y
print(z)

48


In [4]:
x = z = y = "peanutbutter"

In [6]:
club_sandwich = ["bread", "sliced chicken", "bacon", "lettuce", "tomato", "mayonnaise"]

Markdown notes and sentences and analysis written here.

## Question 3

There are 13 markdown cells and 5 code cells in the default template.

## Question 4

In [9]:
import pandas as pd

In [11]:
disney = pd.read_csv("/anvil/projects/tdm/data/disney/flight_of_passage.csv")
disney

date             datetime  SACTMIN  SPOSTMIN
0       05/26/2017  2017-05-26 09:06:38      NaN    -999.0
1       05/26/2017  2017-05-26 09:10:11     47.0       NaN
2       05/26/2017  2017-05-26 09:10:12      NaN       5.0
3       05/26/2017  2017-05-26 09:17:09      NaN      60.0
4       05/26/2017  2017-05-26 09:24:07      NaN      60.0
...            ...                  ...      ...       ...
184813  12/28/2021  2021-12-28 19:35:47      NaN      90.0
184814  12/28/2021  2021-12-28 19:42:15      NaN      90.0
184815  12/28/2021  2021-12-28 19:48:15      NaN      90.0
184816  12/28/2021  2021-12-28 19:54:10      NaN      90.0
184817  12/28/2021  2021-12-28 19:58:35      NaN      80.0

[184818 rows x 4 columns]

In [12]:
disney.head()

date             datetime  SACTMIN  SPOSTMIN
0  05/26/2017  2017-05-26 09:06:38      NaN    -999.0
1  05/26/2017  2017-05-26 09:10:11     47.0       NaN
2  05/26/2017  2017-05-26 09:10:12      NaN       5.0
3  05/26/2017  2017-05-26 09:17:09      NaN      60.0
4  05/26/2017  2017-05-26 09:24:07      NaN      60.0

In [13]:
disney.tail()

date             datetime  SACTMIN  SPOSTMIN
184813  12/28/2021  2021-12-28 19:35:47      NaN      90.0
184814  12/28/2021  2021-12-28 19:42:15      NaN      90.0
184815  12/28/2021  2021-12-28 19:48:15      NaN      90.0
184816  12/28/2021  2021-12-28 19:54:10      NaN      90.0
184817  12/28/2021  2021-12-28 19:58:35      NaN      80.0

There are 184,818 rows and 4 columns in the dataset.

## Question 5

In [14]:
# code here

Markdown notes and sentences and analysis written here.

## Pledge

By submitting this work I hereby pledge that this is my own, personal work. I've acknowledged in the designated place at the top of this file all sources that I used to complete said work, including but not limited to: online resources, books, and electronic communications. I've noted all collaboration with fellow students and/or TA's. I did not copy or plagiarize another's work.

> As a Boilermaker pursuing academic excellence, I pledge to be honest and true in all that I do. Accountable together – We are Purdue.